### ISSUE TITLE
Survey Tools and the End: Very few blocks are analyzable, and possibly no deposits?

### ISSUE THREAD

**Issue by TailedMute** [EnigmaticaModpacks/Enigmatica6#3960]
### Modpack Version

Enigmatica 6 0.5.21

### Describe your issue.

I've gone off and attempted to scan around for mineral deposits with the Mineral Survey Tools, but for some reason, most blocks in the End don't appear to grant the ability to give you a pointer or direction to mineral deposits - and even if they do (end stone sand, trees and dirt, Umbrella Tree "leaves"), it all seems to read as "no mineral deposits."

I'm not exactly willing to go through an entire hour scanning as many islands as possible, with so few blocks even, for the second part: There doesn't seem to be any deposits to find, ever. More specifically, the ones you can find in the End: The Abyssalite and the Prismatic Xylorentinite. Equally so, I'm not really certain about using Core Sample Drills to gather as many samples as possible to figure out if the kits are bugged for the End or simply don't work as expected.

tl;dr: Mineral Survey Tools can be used on few blocks in the End, and Mineral Deposits might not be existent.

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by MuteTiefling** [comment]
It's pretty normal that they work on a limited set of blocks. The same is true in the overworld. We can look to see if that list can be expanded though, to at least include the various ground covers from BYG.

As for whether they spawn at all... That will need review/testing. We can adjust how common they are, if memory serves me. It may just be that they're too rare to commonly find on the actual islands, whereas if you were to scan the void as well you may find them to be more common. Will have a look when I'm able.

**Comment by TailedMute** [comment]
I feel that increasing the list of blocks that can be used in surveys SHOULD be done, of course. It might not be a case of rarity, but rather impossible to find naturally. That would mean Myceliums, Grass, Moss and Nyliums would have to be added, along with End Stone itself...
But after testing, if it does appear to be a little too rare, it may be best to update that, too.
Thank you, in the meantime!

**Comment by TailedMute** [comment]
Update: Checked an Ether Island just now, being that Ether Phylum is in the list of few blocks you can survey.
Everything said "no mineral traces."
It might be extremely rare, or just really bad luck, considering the size of the island.

**Comment by MuteTiefling** [comment]
Looks like survey blocks are hard coded: https://github.com/BluSunrize/ImmersiveEngineering/blob/05fec13c1c5a4e7a3f37043877339e1499df4aa3/src/main/java/blusunrize/immersiveengineering/common/items/SurveyToolsItem.java#L53

After some testing, it appears these just aren't compatible with the End... My first hunch was that BetterEnd may have been interfering but I can't find any with commands even with that disabled. 

Considering these were just a fun extra, I'm afraid they'll just have to go. Will see about moving some of these things into the PNC Amadron Surveys instead.

**Comment by TailedMute** [comment]
Well, that sucks! Amadron Surveys are probably best, then. Really sad we couldn't excavate to our hearts' contents.
